src/Integration.Vsix.UnitTests/SonarLintTagger/TextBufferIssueTrackerTests.cs

```diff
@@ -61,7 +63,8 @@ public void SetUp()
             mockAnalyzerController = new Mock<IAnalyzerController>();
             issuesFilter = new Mock<IIssuesFilter>();
             taggerProvider = CreateTaggerProvider();
-            mockedJavascriptDocumentFooJs = CreateDocumentMock("foo.js");
+            mockDocumentTextBuffer = CreateTextBufferMock();
+            mockedJavascriptDocumentFooJs = CreateDocumentMock("foo.js", mockDocumentTextBuffer.Object);
```

Comment:
Minor change to the test setup: we need access to the mock text buffer so we can simulate it raising an event.